node/core/chain-selection/src/tests.rs

```diff
@@ -2103,10 +2103,7 @@ fn dispute_reversion_against_finalized_is_ignored() {
 		// Sending dispute conculded against message
 		virtual_overseer
 			.send(FromOrchestra::Communication {
-				msg: ChainSelectionMessage::DisputeConcludedAgainst(
-					finalized_number,
-					finalized_hash,
-				),
+				msg: ChainSelectionMessage::RevertBlocks(Vec::from({(finalized_number, finalized_hash)})),
```

Comment:
There should be some tests for sending multiple blocks in `RevertBlocks`, now that it can take more than one.